contracts/liquidity_hub/pool-network/terraswap_pair/src/contract.rs

```diff
@@ -69,6 +83,7 @@ pub fn instantiate(
             deposits_enabled: true,
             swaps_enabled: true,
         },
+        cosmwasm_pool_interface: Addr::unchecked(""),
```

Comment:
Set it to empty from the start, as we need to know the pool contract addr to instantiate the interface.